<div id="sectionA">
    <h1>Introduction</h1>

    


    <p>
        As the project idea was formed, that is, reducing the cost of cultivated meat by optimizing the growth factor FGF2, it became clear that the dry lab would become an important part of the process. It was believed that our approach to improve the binding affinity between FGF2 and its receptor FGFR2 by site directed mutagenesis, would be greatly benefited from in silico modeling. The idea being that if we could evaluate the effect of mutations computationally, we would be able to pinpoint mutations that are more likely to work. This is useful due to the fact that there are 20 possible natural amino acids for each position within a given protein chain. To produce mutant proteins in the lab for every possible amino acid in each position is an unreasonable amount of work, and without any guarantee of success.
        At the beginning we explored different softwares available to predict mutation effects in silico but we wanted to be sure about our results. Therefore, we consulted with experts in this field at Uppsala University and they suggested using free energy perturbation (FEP) through the molecular dynamics software package Q. With Q, we were able to estimate the effect of single point mutations computationally. This allowed us to proceed in the lab with the single point mutations that had satisfactory in silico estimation results, thus saving us extensive lab work and increasing the chance of successful results.    
        </p>
        




    <h2>Structure selection</h2>
    <p>
        Our project initially started with research using UniProt and the Protein Data Bank (PDB) to find sequences and crystal structures of the bovine growth factor FGF2, and its receptor bovine FGFR2.There was no crystal structure for bovine FGF2 available, only for human FGF2. To see if the bovine protein may have a similar 3D structure, we performed a sequence alignment using Clustal Omega to assess the homology of the two proteins. As the sequences were found to have very high sequence identity - they only differed by two residues. The crystal structure of human FGF2 bound to human FGFR2 was chosen as the target structure. The crystal structure for human FGF2 was visually examined in PyMOL to search for plausible mutation sites. Furthermore, homology modeling of bovine FGF2 with human FGF2 as template on Swiss-Prot showed that the 3D structure of bovine FGF2 was very likely to be nearly identical to the 3D structure of human FGF2 which gave further support for the choice of the human FGF2 crystal structure as a target to identify mutations. The human FGF2 protein and receptor structure can be seen in Figure 1. 
    </p>
    <div class="image middle half">
        <img class="expandable_image" width="1564" height="955" src="https://2021.igem.org/wiki/images/6/67/T--Uppsala--FGF2andFGFR2.png">
    <p> Figure 1. Structure used to identify plausible mutations. PDB code 1EV2. </p>    
    </div>

        


    <h2>Initial mutation pool</h2>
<p>
The search for mutations was conducted in the GUI in pyMOL. We explored the binding site of FGF2 with its receptor and analysed the existing interactions to see if we could find holes in the structure which we could fill by inserting larger residues. By comparing the amino acid sequence of FGF2 to that of other FGFs, differences in the binding site were identified and the possible effect these alternative residues could have in FGF2 were investigated. We also looked for natural or synthetic mutations that had been found in analogous structures to FGF2 in existing literature. We wanted to improve the existing interactions and to mutate residues to create new interactions. Some sites were excluded directly because of high side chain clash scores in PyMOL. The resulting first mutation pool, consisting of positions that seemed likely to be apt for improving mutations, is presented here:
</p>

  

</div>
<div id="sectionD">
    <h1>Filtering of mutations</h1>
    <div>

    <div class="image left half">
      <img class="expandable_image" width="286" height="536" src="https://2021.igem.org/wiki/images/2/27/T--Uppsala--filter-crop.png">
  <p> Figure 2. Different tools used to evaluate and filter mutations </p>    
  </div>
<p>
Optimally, all sites within the mutation pool would be evaluated using molecular dynamics simulations. However, as free energy perturbation is computationally demanding and our computing power resources were limited, other means of evaluating the mutations were used in order to reduce the size of the mutation pool (Figure 2). These included: <a href="http://foldxsuite.crg.eu/">FoldX</a> through the GUI In <a href="http://www.yasara.org/index.html">Yasara</a>, the residue scanning tool within the Maestro suite, docking on cluspro and evaluation in Dynamut. In Yasara, the PDB-file of human FGF2 in complex with FGFR2 was loaded and prepared using a repair module within the FoldX suite. Then, a residue scanning tool within the FoldX suite was used to evaluate the effect of mutating the remaining nineteen natural amino acids. In maestro, the structure was loaded and repaired similarly, but the residue scanning tool within the Maestro suite gave information on the effect of both affinity and stability. For ClusPro, the structure of FGFR2 and differently mutated versions of FGF2 were submitted for docking. For ClusPro, the structure of FGFR2 and differently mutated versions of FGF2 were submitted for docking. The calculated change in Gibbs free energy (ΔG) of the suggested binding mode was chosen and compared to the wild type. All combined, this resulted in a reduction of the mutation pool from 36 to a size of 8 mutations that were to be evaluated computationally using molecular dynamics through the molecular dynamics software <a href="https://github.com/qusers/qligfep">package Q</a> developed at Uppsala University.
</p>
</div>


<h2> SAAFE-SEQ </h2>
<p> 
  SAAFE-SEQ [1] uses a machine learning algorithm to predict the change in stability of a protein upon single mutation. For each mutation, it combines the sequence information and  amino acid properties to determine the change in free energy to predict the stability. The change in  Gibbs free energy provides us with the information on whether the mutations are stabilizing or destabilizing the protein structure. It takes our wild type protein fasta sequence and mutation position, wild type and mutant amino acid as input. 
</p>

<h2>Dynamut</h2>
<p>
  Most of the software tools use static structure based approaches for assessing the impact of mutations on protein structure because of high computational costs. As proteins are highly mobile molecules we used Dynamut [2] which visualizes and analyzes protein dynamics.It estimates the impact of mutations on protein dynamics and stability resulting from vibrational entropy changes.
</p>

<h2>Cluspro</h2>
<p>
  Cluspro [3] uses the PDB codes as input for protein-protein docking, and returns a ranked list of complexes, ordered by clustering properties. We worked with PDB files of FGF2 and the receptor FGFR, then used ClusPro to dock them which resulted in a change in free energy values. After that, we mutated the FGF2 and docked that with FGFR to get the change in free energy values of the mutated complex. Finally we compared the stability of the formed mutant complex with wild type docking pose by analyzing the values returned by docking the wild type and the mutant. 
</p>

<h2>FoldX</h2>
<p>
  FoldX [4] estimates the vital interactions contributing to the stability of proteins as well as complexes. It distinguishes itself from other software by using full atomic descriptions. It takes various energy terms into account, including differences in solvation energy for apolar and polar groups, free energy of inter- and intramolecular hydrogen bonds,as well as electrostatic contributions of charged groups. By installing the FoldX plugin for YASARA, the FoldX tools can be accessed  in the 3D graphical YASARA interface. We used the position scanning module within FoldX in order to get the results in Table 1.
</p>

<table>
    <p style="padding-bottom:0;">Table 1. ΔΔGibbs free energy from FoldX.</p>
    <tr>
      <th>Position</th>
      <th>Lowest (kcal/mol)</th>
      <th>Second lowest (kcal/mol)</th>
      <th>Thrid lowest</th>
    </tr>
    <tr>
      <td>M142X</td>
      <td><b>M142L</b> 0.38926</td>
      <td><b>M142I</b> 0.791585</td>
      <td><b>M142N</b> 1.32906</td>
    </tr>
    <tr>

            <td>P141X</td>
            <td><b>P141L</b> 1.34137</td>
            <td><b>P141V</b> 1.6697</td>
            <td><b>P141I</b> 1.67707</td>

    </tr>
    <tr>

    <td>G28X</td>
    <td><b>G28A</b> 5.35588</td>
    <td><b>G28S</b> 6.6389</td>
    <td><b>G28C</b> 8.16446</td>

    </tr>
    <tr>
        <td>Q54X</td>
        <td><b>Q54R</b> -2.39739</td>
        <td><b>Q54I</b> -1.28936</td>
        <td><b>Q54K</b> -1.27196</td>
    </tr>
    <tr>
        <td>L98X</td>
        <td><b>L98I</b> 0.585657</td>
        <td><b>L98M</b> 0.641692</td>
        <td><b>L98P</b> 0.94977</td>
    </tr>
    <tr>
        <td>F93X</td>
        <td><b>F93W</b> -0.263676</td>
        <td><b>F93L</b> 0.559059</td>
        <td><b>F93Y</b> 0.614088</td>
    </tr>
    <tr>
        <td>V88X</td>
        <td><b>V88I</b> -0.393335</td>
        <td><b>V88L</b> 0.30579</td>
        <td><b>V88M</b> 0.409882</td>

      </tr>

      <tr>
        <td>Q56X</td>
        <td><b>Q56M</b> -0.789795</td>
        <td><b>Q56K</b> 0.273197</td>
        <td><b>Q56I</b> 0.399764</td>
      </tr>
      <tr>

        <td>E58X</td>
        <td><b>E58L</b> -1.52055</td>
        <td><b>E58I</b> -1.19611</td>
        <td><b>E58F</b> -1.18686</td>

      </tr>

      <tr>
        <td>K44X</td>
        <td><b>K46P</b> -1.03922</td>
        <td><b>K46E</b> -0.647498</td>
        <td><b>K46D</b> -0.547631</td>
      </tr>

  </table> 
  
  <h2>Residue Scanning</h2>

  <p>
    We used the residue scanning panel in Maestro from Schrödinger [5] .The panel helped us to scan our proteins for residues that are suitable for mutations based on solvent accessible surface area and distances with other residues and also allows manual selection.To produce mutated structures,we used the mutations based on the findings from PyMOL. As a result, it returned the relative change in thermal stability of a single protein as well as its binding affinity compared to the original protein as mentioned in Table 2, which helped us to move further in the process of evaluating the mutations.
    </p>

    <table>
        <p style="padding-bottom:0;">Table 2. Δ affinity and stability values from residue scanning. Negative values are marked in green, indicating increased affinity or stability, and thus potential mutations for further evaluation.</p>
        <tr>
          <th>MUTATIONS</th>
          <th>Δ AFFINITY (kcal/mol)</th>
          <th>Δ STABILITY (kcal/mol)</th>
        </tr>
        <tr>
          <td>M142I</td>
          <td>5.156</td>
          <td>0.359</td>
        </tr>
        <tr>
            <td>M142N</td>
            <td>7.29</td>
            <td>16.33</td>
    
        </tr>
        <tr>
            <td>P141L</td>
            <td>-1.398</td>
            <td>8.850</td>
    
        </tr>
        <tr>
            <td>P141V</td>
            <td>0.581</td>
            <td>2.787</td>
        </tr>
        <tr>
     
                <td>P141I</td>
                <td>-3.309</td>
                <td>5.798</td>
         
        </tr>
        <tr>
            <td>G28A</td>
            <td>0.028</td>
            <td>6.492</td>
        </tr>
        <tr>
            <td>G28S</td>
            <td>-0.656</td>
            <td>9.946</td>
          </tr>
    
          <tr>
            <td>Q54R</td>
            <td>-7.796</td>
            <td>0.526</td>
          </tr>
          <tr>
            <td>Q54I</td>
            <td>1.093</td>
            <td>2.929</td>
          </tr>
    
          <tr>
            <td>Q54K</td>
            <td>-4.074</td>
            <td>11.392</td>
          </tr>


          <tr>
            <td>L98I</td>
            <td>0.260</td>
            <td>2.667</td>
          </tr>

          <tr>
            <td>L98M</td>
            <td>-7.205</td>
            <td>2.066</td>
          </tr>

          <tr>
            <td>F93W</td>
            <td>-1.303</td>
            <td>10.466</td>
          </tr>

          <tr>
            <td>F93L</td>
            <td>-1.906</td>
            <td>1.047</td>
          </tr>

          <tr>
            <td>F93Y</td>
            <td>-0.121</td>
            <td>-0.672</td>
          </tr>

          <tr>
            <td>V88I</td>
            <td>-1.298</td>
            <td>-3.134</td>
          </tr>

          <tr>
            <td>V88L</td>
            <td>6.29</td>
            <td>-1.070</td>
          </tr>
    
          <tr>
            <td>V88M</td>
            <td>-3.456</td>
            <td>7.171</td>
          </tr>

          <tr>
            <td>Q56M</td>
            <td>9.456</td>
            <td>-7.723</td>
          </tr>

          <tr>
            <td>E58L</td>
            <td>3.518</td>
            <td>-0.972</td>
          </tr>

          <tr>
            <td>E58I</td>
            <td>0.985</td>
            <td>-0.039</td>
          </tr>

          <tr>
            <td>E58F</td>
            <td>1.046</td>
            <td>4.899</td>
          </tr>

          <tr>
            <td>K46E</td>
            <td>2.134</td>
            <td>-12.453</td>
          </tr>

          <tr>
            <td>K46D</td>
            <td>0.782</td>
            <td>-7.769</td>
          </tr>

          <tr>
            <td>K46P</td>
            <td>0.602</td>
            <td>-8.445</td>
          </tr>

      </table>
      <p>
        Based on the above filtering process, we constructed a pool of potential mutations which were found worthy of further evaluation using molecular dynamics through the molecular dynamics software package Q. The pool of 8 mutations:
      </p>
      
      <p style="background-color: #426968; margin-left: 50%; width: 40vw; font-style: italic; color: white; font-size: 1.5em; text-align: center; transform: translateX(-50%); border-radius: 3vh; margin-bottom: 3vh; padding-top: 2vh;">
        Q54K &nbsp;&nbsp;&nbsp;&nbsp;Q54R&nbsp;&nbsp;&nbsp;&nbsp;Q56M &nbsp;&nbsp;&nbsp;&nbsp;V88I <br>
        F93Y &nbsp;&nbsp;&nbsp;&nbsp;F93L&nbsp;&nbsp;&nbsp;&nbsp;F93W &nbsp;&nbsp;&nbsp;&nbsp;L98M <br>
        </p>


          <h2>Tools Used</h2>
          <p> <a href="http://compbio.clemson.edu/SAAFEC-SEQ/#started">SAAFE-SEQ</a></p>
          
          <p>
            <a href="http://biosig.unimelb.edu.au/dynamut/prediction">DynaMUT</a>
          </p>
          
          <p>
            <a href="http://foldxsuite.crg.eu/">FoldX</a>
          </p>
          <p>
            <a href="https://cluspro.org/publications.php">ClusPro</a>
          </p>
          
          <p>
            <a href="https://www.schrodinger.com/sites/default/files/s3/mkt/Documentation/current/docs/Documentation.htm#bioluminate_help/residue_scanning.html?Highlight=residue%20scanning">Residue Scanning in Maestro from Schrodinger</a>
          </p>
          
          <p>
            <a href="https://openwetware.org/wiki/UCSF_Chimera_for_3D_Printing">Chimera</a>
          </p>

</div>

<div id="sectionE">
  <h1>3D printing</h1>

<p>    
  Having the opportunity to use U-Print, a dedicated facility for 3D printing, at Uppsala university, inspired the idea of 3D printing the structure of FGF2. For such prototyping we needed to create a file that described the surface geometry of a three dimensional object.To generate such a file (Figure 3), in the STL format, we used Chimera [6], a software used for interactive visualization and analysis of molecular structure. 
</p>


<div class="image middle half">
  <img class="expandable_image" width="800" height="369" src="https://2021.igem.org/wiki/images/thumb/1/1c/T--Uppsala--merged-mut-sel-f3.png/320px-T--Uppsala--merged-mut-sel-f3.png">
<p>Figure 3. Solid Surface file of FGF2</p>    
</div>


<p>
  The stereolithography technique was used to 3D print both the protein and the receptor. Research Engineer Olle Erksson from U-Print printed our model and helped us to get to know the process behind the technology. He also gave us a small guided tour of their facility, which included 3D printed anatomical models for surgery, tools for bacterial culture, tools for live imaging, protein structures, etc.
  </p>


  <div>

    <div class="image left half stacked">
      <img class="expandable_image" width="1024" height="682" src="https://2021.igem.org/wiki/images/4/4f/T--Uppsala--fgf.jpg">
    <p> Figure 4. Structure of Fibroblast Growth Factor 2. 
      <i>3D printing was performed at U-PRINT: Uppsala University’s 3D-printing facility at the Disciplinary Domain of Medicine and Pharmacy.</i>
       </p>    
    </div>
    
    <div class="image right half stacked">
      <img class="expandable_image" width="1024" height="892" src="https://2021.igem.org/wiki/images/thumb/a/a9/T--Uppsala--recept.jpg/600px-T--Uppsala--recept.jpg.png">   
    
      <p> Figure 5. Structure of Fibroblast Growth Factor receptor. 
        <i>3D printing was performed at U-PRINT: Uppsala University’s 3D-printing facility at the Disciplinary Domain of Medicine and Pharmacy.</i>
         </p>  
    </div>

    </div>


    <p>
      We wanted to explore and educate ourselves with different technologies and tried to integrate them into our project. Visual representation of FGF2 is one of the important benefits for creating a 3D model of our protein. In addition to the protein structure, U-Print helped us to print the receptor structure (Figure 4, Figure 5). 

      </p>

<h2> References</h2>
<p>
  [1] SAAFEC-SEQ: A Sequence-Based Method for Predicting the Effect of Single Point Mutations on Protein Thermodynamic Stability. DOI: <a href="https://www.mdpi.com/1422-0067/22/2/606">10.3390/ijms22020606</a>
</p>

<p>
  [2] DynaMut: Predicting the impact of mutations on protein conformation, flexibility and stability. PMID: 29718330 PMCID: PMC6031064 DOI: 10.1093/nar/gky300
</p>
<p>
  [3] Desta IT, Porter KA, Xia B, Kozakov D, Vajda S. Performance and Its Limits in Rigid Body Protein-Protein Docking. Structure. 2020 Sep; 28 (9):1071-1081.  doi

  Vajda S, Yueh C, Beglov D, Bohnuud T, Mottarella SE, Xia B, Hall DR, Kozakov D. New additions to the ClusPro server motivated by CAPRI. Proteins: Structure, Function, and Bioinformatics. 2017 Mar; 85(3):435-444.pdf
  
  Kozakov D, Hall DR, Xia B, Porter KA, Padhorny D, Yueh C, Beglov D, Vajda S. The ClusPro web server for protein-protein docking. Nature Protocols. 2017 Feb;12(2):255-278.pdf
  
  Kozakov D, Beglov D, Bohnuud T, Mottarella S, Xia B, Hall DR, Vajda, S. How good is automated protein docking? Proteins: Structure, Function, and Bioinformatics. 2013 Dec; 81(12):2159-66.pdf
</p>

<p>
  [4] Joost Schymkowitz, Jesper Borg, Francois Stricher, Robby Nys, Frederic Rousseau, Luis Serrano, The FoldX web server: an online force field, Nucleic Acids Research, Volume 33, Issue suppl_2, 1 July 2005, Pages W382–W388, <a href="https://academic.oup.com/nar/article/33/suppl_2/W382/2505499">https://doi.org/10.1093/nar/gki387</a>
</p>

<p>
  [5] Schrödinger Release 2021-3: BioLuminate, Schrödinger, LLC, New York, NY, 2021.
</p>

<p>
  [6] Molecular graphics and analyses performed with UCSF Chimera, developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco, with support from NIH P41-GM103311.
<a href="https://pubmed.ncbi.nlm.nih.gov/15264254/">UCSF Chimera--a visualization system for exploratory research and analysis.</a> Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, Ferrin TE. J Comput Chem. 2004 Oct;25(13):1605-12.
</p>

</div>
