{{template_top.html}}

            <!-- page title-->
            <div id=parallaxtop style="background-image: url('https://2021.igem.org/wiki/images/1/17/T--Uppsala--newnotebookbanner.png'); background-attachment: local;"></div>



            <!--Sidebar-->
            <div id="sidebar">
                <div id="sticky_sidebar">
                    <div id="sidebar_header">
                        <a href="#pagecontent">
                            Notebook
                        </a>
                    </div>
                    <hr>
                    <ul id="sidebar_list_container">
                        <!-- template for a single entry in this list. the real entries will be added at runtime
                            <li class="sidebar_list_section">
                                <span id="span1">
                                    <a href="#section1_link">
                                        Lab Safety
                                    </a>
                                </span>
                            </li>
                        -->
                    </ul>
                </div>
            </div>

            <!--Actual content-->
            <div id="fgfuture_content">

                <!-- INSERT GOOGLE DOC CONTENT BELOW THIS LINE -->
<div class="content_section" id="labintro">
<h1> Lab Introduction </h1>

<p>This year's iGEM journey began in February for the Uppsala team. During that time a virus, tiny in size but huge in impact, made it impossible for us to meet each other in real life. Instead we had to make due with seeing each other over zoom. As we were getting to know each other online we were also spawning and researching project ideas. We voted for a group of ideas and a selection of 6 ideas from 25 in total, were chosen to be further researched in groups. Together in these groups we wrote reports and made presentations of our ideas. After the presentations we got feedback from professors and faculty at Uppsala University. With that as the basis we made a final vote and decided that reducing the cost of cultivated meat through enzyme engineering was our project. After the decision we all started to work towards the same goal and we were impatiently waiting to finally enter the lab.</p>

<p>
During the summer iGEM Uppsala worked in three different lab groups with three different approaches. Lab group 1 wanted to express the wild type FGF2 and mutate it to increase binding affinity. Lab group 2 had a mission to create a chimeric protein of FGF1 and FGF2 which is called FGFC and lab group 3 wanted to express a hyperstable variant of FGF2. During the first two weeks and also the last weeks we worked together at BMC since we all had similar laboratory tasks to do. Also during one and a half weeks of August we had the opportunity to work at TESTA Center in order to upscale our project.
</p>
<p>
This is a summary of every lab week for iGEM Uppsala 2021. For more detailed information about the laboratory practices see <a href="https://2021.igem.org/Team:Uppsala/Experiments">Experiments</a>.
</p>
</div>
<div class="content_section" id="section1">
<h1> Week 1-2 (7 June - 20 June)</h1>
<h2>Team BMC</h2>

<p>
The team finally entered the lab Uppsala Biomedical Centre (BMC), but first and foremost iGEM Uppsala had a tour of the lab and a safety introduction in order to prevent accidents. The first week at BMC mainly focused on planning for the summer and creating appropriate protocols. In terms of actual lab work the team prepared LB liquid media and LB agar plates with chloramphenicol to practice for the weeks ahead.
</p>

<p>
Second week also consisted of practicing important laboratory techniques. We transformed plasmids into competent DH5alpha cells. We made more LB agar plates with kanamycin and ampicillin and followed protocols on making DH5alpha competent cells. The iGEM Uppsala team practiced streaking for single colonies on the transformed cells. There was also designing and ordering of primers and planning regarding lab cleaning. 
</p>


</div>
<div class="content_section" id="section2">
<h1> Week 3-4  (21 June - 4 July)</h1>

<h2> Labgroup 1 </h2>
<p> We transformed wild type FGF into DH5alpha cells. Afterwards we amplified the plasmid by using PCR. Lastly we purified the PCR product and sent it for sequencing. </p>
<p> During the fourth week the sequencing results were analyzed and they were satisfactory, which enabled us to start the purification of the plasmid. We performed PCR to add the terminator to the FGF2 plasmid. Labgroup 1 also tried to digest the pET vector and  ligate it with FGF2. Due to low concentrations we redid it a few times. Finally we sent them for sequencing. </p>

<h2> Labgroup 2 </h2>
<p>During week 3 we performed transformation of the BioBrick pET plasmid into TOP10 competent cells. Then we performed transformation of the plasmid with
FGFC. We also confirmed the transformation with PCR and gel
electrophoresis. </p>
<p>The following week we did plasmid prep with several purifications. We tried to confirm transformation with PCR and gel electrophoresis but was unsuccessful twice. We did transformations but unfortunately with no cell growth. We were hoping to send sequencing but after observing the agar plates without any colonies, it was not an option. Instead new overnight cultures were made and the procedure was started once more. </p>

<h2> Labgroup 3 </h2>
<p> Lab group 3 made a transformation of the BioBrick pET plasmid into TOP10 competent cells and also transformed the IDT plasmid containing the hyper stable FGF2 into DH5alpha competent cells. The results of the two transformations were verified using colony PCR and running the amplified products on a gel electrophoresis. The PCR products also got sent to sequencing and glycerol cell stocks were made of the DH5alpha competent cells with the plasmid inserted.
</p>
<p> The fourth week lab group 3 purified the pET plasmid and the IDT plasmid using plasmid miniprep kits. We also had multiple attempts of adding the terminator to the hyper stable FGF2 by using PCR. After each PCR the samples were tested on a gel electrophoresis but with no success. Lastly alternative approaches on how to add the terminator that might work better were planned.</p>
</div>

<div class="content_section" id="section3">
<h1> Week 5-6 (5 July - 18 July) </h1>

<h2> Labgroup 1 </h2>
<p> This week labgroup 1 redid digestion and ligation and ran the product on a gel. Then overnight cultures were grown and pET plasmid from a band on the gel electrophoresis was cut out and purified. Next step was transformation of ligated pET and wild type FGF2 was also transformed into DH5alpha. A colony PCR was performed, plasmids were purified using kits and more attempts of digestion using NdeI and SpeI were made.
</p>
<p>We continued to try to digest and ligate the biobrick parts, but with limited success. Therefore we scheduled a meeting with Anthony Foster and came to the conclusion that we need to add base pairs outside the restriction sites on the FGF2 primers in order for the restriction enzymes to be able to cut the DNA. Therefore lab coordinators and co-coordinators designed new primers with 6bp overhangs which arrived the following monday. </p>

<h2> Labgroup 2 </h2>
<p> This week started with trying to confirm that we had amplified the FGFC-terminator by using an agarose gel electrophoresis. It seemed to have failed with no clear bands, therefore the PCR to add terminator to FGFC was repeated. It wasnâ€™t successful at this point as well which made us repeat the previous steps once again. After that one sample on the gel showed promising results and we went on with PCR purification. However, a new PCR was also performed with a gradient in order to get an idea of what the most optimal annealing temperature could be. From this point, we continued with DpnI digestion of the FGFC-term construct to get our gene without methyl groups. </p>
<p> The next week the DpnI digested FGFC-term went on to be digested with the restriction enzymes NdeI and SpeI separately. This was followed by ligation into the PET-vector and transformation in DH5alpha. Colony PCR was run the day after and observed in gel electrophoresis, with no visible bands therefore PCR was repeated once more. FGFC-term was double digested with the same restriction enzymes, followed by ligation. Transformation was repeated and in the following day, the colonies were purified which showed very low DNA concentration. We went back to the first step and purified the PUC-vector. </p>

<h2> Labgroup 3 </h2>
<p> Week 5 mainly consisted of continuing to try to add the terminator to the hyper stable FGF2 gene and testing different PCR programs and fresh reagents in order for the PCR reaction to work. Finally we tested with a new DNA polymerase with fresh buffer and nucleotides and managed to successfully add the terminator with the correct restriction sites.
</p>
<p> The other week we started by purifying the plasmid that contained the FGF2hs and the terminator. Then we continued with digestion in order to remove a piece of the insert to try to complete our construct. Afterwards we tested the results by running the product on a gel electrophoresis. Then we purified the pcr product and sent it for sequencing. 
</p>
</div>

<div class="content_section" id="section4">
<h1> Week 7-8 (19 July - 1 August) </h1>
<h2> Labgroup 1 </h2>
    <p>
    Week 7 started off by adding 6 base pair overhangs to the FGF2 with the newly designed primers. This was done by using the old FGF2 as a template when using the new primers in the PCR.
    It was sent for sequencing and the result was promising. We therefore started to digest and ligate it with the pET vector. This time it worked excellent on the first try, compared to the two weeks before, where the experiment continually failed. They were sent to sequencing that friday. 
    </p>
    <p> The following week the sequencing result came back correctly! Therefore we started the mutagenesis project. We created 4 different mutations, FGF2 Q54K, FGF2 V88I, FGF2 L98M and FGF2 Q54K + L98M. These were all created using a mutagenesis protocol designed by Anthony Foster. Primers for mutagenesis were used in a PCR to create these mutations. For the double mutation, FGF2 Q54K was exposed to the FGF2 L98M mutation protocol. All mutations were sent for sequencing and everything was correct! Afterwards we created glycerol cell stocks. </p>


<h2> Labgroup 2 </h2>
    <p>
    We decided to divide into two groups. The first group did a PCR with new overhangs that  was designed the week before. The old FGFC sequence was used as the basis for this experiment. PCR purification was performed, however the final DNA concentration was  insufficient. <br>
    The other group did a PCR on FGFC and the terminator separately. These were then digested with PstI. This made them compatible with ligating to each other, which gave a sufficient DNA concentration of the wanted product. Lastly the ends of the construct were cut with NdeI and SpeI. </p>
    <p> The next week started off a colony PCR. Then we continued with our plan to add the terminator as an overhang on the primers and use PCR. After the PCR, the samples were put on a gel electrophoresis and the samples were purified by using a kit. DpnI digestion was performed and from the gel electrophoresis a band was extracted in order to purify the plasmid. Finally we performed transformations into BL21 cells for our FGFC.  </p>

<h2> Labgroup 3 </h2>
    <p>
        This week we used new primers that had 6 added base pairs on each end of the hyper stable FGF2-Terminator sequence. We ran the results on a gel to verify the results. We also tried to do a single digestion, which means that we first cut with NdeI and then purify and then cut with SpeI. Afterwards we ligated the product into the pET vector. However due to low yield we instead did a double digestion and then purified and ligated it into the pET vector. Afterward we transformed the cells and made overnight cultures and a colony PCR to see if the bacteria had our gene of interest. This was verified with a gel electrophoresis. Lastly we purified our samples with a purification kit and sent them for sequencing.
    </p>
    <p> 
        Last week we sent the final construct for sequencing and the sequencing results were great! Therefore overnight cultures were made and we purified the plasmids using a plasmid purification kit. The plasmids were transformed into a BL21 E.coli strain for both the wild type FGF2 and the Hyperstable FGF2. Afterwards we let the transformed cells grow on agar plates and performed Colony PCR. We then purified the PCR samples for both the FGF2 wt and the FGF2hs and sent them to sequencing. Towards the end of the week we made glycerol cell stocks in order to have the proper strain with the correct construct that could be used to produce FGF2.
        </p>
</div>

<div class="content_section" id="section5">
<h1> Week 9-10 (2 August - 15 August)</h1>

    <h2>Team TESTA </h2>
    <p>
        For two days this week and the upcoming week six members of our team got the opportunity to work at TESTA Center in order to scale up our project! The work started with transformation of all our FGF2 plasmid variants into E. coli BL21 strains. Then it was time to grow bacteria in small E-flasks and induce the cells with IPTG to test optimal conditions to express our different FGF2 variants. Afterwards we lysed the cells and ran an SDS PAGE.
    </p>
        <p>
            Week 10 at Testa Center started with preparation of the two bioreactors each with a total volume of 2.5L media that we are using to express our protein on a larger scale. We picked the hyper stable FGF2 and the wild type FGF2 to grow in the bioreactors since the team at BMC had the most success with these two variants. After the cells had grown in the bioreactor we harvested the cells then we lysed them by using a french press and finally used IMAC to purify our protein. The purified protein was tested on an SDS PAGE to determine how successful our purification was.
        </p>

    <h2>Team BMC </h2>
        <p>
        During this week the people over at BMC expressed all FGF2 variants and visualized the results by running samples on an SDS PAGE. We also selected colonies which still contained the pLysS plasmid after transformation, which is needed for proper overexpression of FGF2.
        Afterwards we picked FGF2hs and wt FGF2 and continued on with lysing the cells in a french press. Then we purified these samples by using a small IMAC column. The purified samples were then put on an SDS PAGE and we saw clear bands however the bands were slightly shorter than expected. Lastly the BL21 E.coli cells were transformed with several FGF2 variants.
        </p>
        <p> 
            At BMC the following week the team prepared all the buffers necessary for IMAC purification. Then overnight cultures were grown of FGFC, FGFhs, FGF wt and also FGF2 Q54K. The cells were then induced with IPTG at 0.6 OD and grew for 6 hours. Afterwards purification with IMAC was performed and an SDS PAGE was used to verify the overexpression. Lastly we tested to transform our different FGF2 variants into new BL21 competent E. coli cells and expression was performed.
        </p>
</div>

<div class="content_section" id="section6">
<h1> Week 11-12 (16 August - 29 August) </h1>
<h2>Team BMC</h2>
    <p>
        Week 11 began with inoculating overnight cultures and inducing them at OD 0.6 to start the expression of FGF2 for all variants. The samples were then lysed in a french press and the lysed cells were purified using IMAC. Afterwards the samples got placed on an SDS PAGE but no bands were visible. After the purification the samples got spin concentrated in order to remove imidazole and to concentrate the protein. FGF2 was also put on a heparin column in order to look at the activity of the protein and to purify it further. Finally the protein concentrations were determined with Bradford Assay. 
    </p>
        <p>
            Now we have reached the final week of laboratory work and we are ready to finally tie it all together. We started overnight cultures of FGF2 wt, FGFhs, FGFC and FGF2 L98M and induced them at OD 0.6. Then the cells grew for 6 hours and a small sample was taken to run an SDS PAGE to look at the overexpression. The rest of all the samples were lysed in a french press. After the cells were lysed, the protein was purified by using an IMAC nickel column. Then another SDS PAGE was performed to look at the lysate, the flow through from the column, the washed IMAC product and finally the elution. Afterwards all the samples were snap freezed using liquid nitrogen and then placed in a -80 degrees celsius freezer. 
        </p> 
    <p> 
        This marks the end of the lab work which iGEM Uppsala did during the summer of 2021.
    </p>
</div>

                <!-- INSERT GOOGLE DOC CONTENT ABOVE THIS LINE -->

            </div>

{{template_bottom.html}}